templates/default/mysql-job.sh.erb

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# rdiff-backup MySQL job script for $JOB managed by Chef. Changes will be overwritten.
+
+# This script is normally run by cron, but can be run manually to start the job if it is not already running.
+
+FILE=`basename $0`
+DIR=`dirname $0`
+PIDFILE="$DIR/.$FILE.lock"
+JOB="${FILE%.sh}"
+LOGPATH='/var/log/rdiff-backup'
+INSTANCES=`pgrep -fu <%= @suser %> $FILE | wc -l`
+
+# Prepends date to every line of stdin, then writes to both stdout and the log file.
+function log() {
+  sed "s/^/[`date +'%Y-%m-%d %H:%M:%S'`] /" | tee >(/usr/sbin/cronolog "${LOGPATH}/${JOB}/%Y-%m.log")
+}
+
+# Prepends date to every line of stdin, then writes to stdout and both the individual and general log files.
+function logboth() {
+  sed "s/^/[`date +'%Y-%m-%d %H:%M:%S'`] /" | tee >(/usr/sbin/cronolog "${LOGPATH}/${JOB}/%Y-%m.log") >(/usr/sbin/cronolog "${LOGPATH}/general/%Y-%m-%d.log" -S "${LOGPATH}/current.log")
```

Comment:
This is a slightly unusual way to define a function in bash, most of the time you use functions when you do something with the args. You may want to put a comment stating that the function should be called as
```bash
thing_with_output | log{,both}
```

rather than
```bash
log{,both} thing_with_output
```